301-webapp-storage-keyvault-msi/scripts/deploy.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
remove this file, scripts need to be deployable using the scripts in the root of the repo